zeppelin-server/src/test/java/org/apache/zeppelin/integration/ParagraphActionsIT.java

```diff
@@ -269,4 +269,38 @@ public void testDisableParagraphRunButton() throws Exception {
     }
 
   }
+
+  @Test
+  public void testClearOutputButton() throws Exception {
+    if (!endToEndTestEnabled()) {
+      return;
+    }
+    try {
+      createNewNote();
+
+      waitForParagraph(1, "READY");
+      WebElement paragraph1Editor = driver.findElement(By.xpath(getParagraphXPath(1) + "//textarea"));
+      paragraph1Editor.sendKeys("println" + Keys.chord(Keys.SHIFT, "9") + "\""
+          + "abcd\")");
+      collector.checkThat("Before Run Output field contains ",
+          driver.findElement(By.xpath(getParagraphXPath(1) + "//div[contains(@ng-if,'getResultType()')]")).getText(),
```

Comment:
Same reusable xpath advice than in the other PR.
Maybe after those are merged, you could create a PR to apply it in all the tests